src/sinol_make/helpers/package_util.py

```diff
@@ -22,20 +23,28 @@ def get_task_id() -> str:
             util.exit_with_error("Invalid task id. Task id should be 3 characters long.")
 
 
-def extract_test_id(test_path):
+def extract_test_id(test_path, task_id=None):
     """
     Extracts test group and number from test path.
     For example for test abc1a.in it returns 1a.
     :param test_path: Path to test file.
+    :param task_id: Task id. If None, it is extracted from config.yml.
+                    If not found, it's assumed that the length of task id is 3.
```

Comment:
Maybe it would be cleaner to initialize `task_id` somewhere at the beginning of the code, and then only pass a non-None value to the functions?